example/lib/screen/xayn_widgets/page/dots_indicator_page.dart

```diff
@@ -0,0 +1,304 @@
+            child: pageWithDots,
+          ),
+        ),
+        Expanded(
+          flex: 3,
+          child: ListView(
+            physics: const ClampingScrollPhysics(),
+            children: [
+              _buildListTileDotsColor(),
+              _buildListTileDotsActiveColor(),
+              _buildListTileDotsBorderColor(),
+              _buildListTileDotsSize(),
+              _buildListTileDotsBorderWidth(),
+              _buildListTileDotsShape(),
+              _buildListTileDotsSpacing(),
+            ],
+          ),
+        )
+      ],
+    );
+  }
+
+  Widget _buildListTileDotsColor() {
+    return ListTile(
+      title: const Text('Dots color: '),
+      trailing: DropdownButton<_DotsColor>(
+        value: dotsColor,
+        items: _DotsColor.values.map(_itemBuilderDotsColor).toList(),
+        onChanged: (value) {
+          setState(() {
+            dotsColor = value!;
+          });
+        },
+      ),
+    );
+  }
+
+  Widget _buildListTileDotsActiveColor() {
+    return ListTile(
+      title: const Text('Dots active color: '),
+      trailing: DropdownButton<_DotsColor>(
+        value: dotsActiveColor,
+        items: _DotsColor.values.map(_itemBuilderDotsColor).toList(),
+        onChanged: (value) {
+          setState(() {
+            dotsActiveColor = value!;
+          });
+        },
+      ),
+    );
+  }
+
+  Widget _buildListTileDotsBorderColor() {
+    return ListTile(
+      title: const Text('Dots border color: '),
+      trailing: DropdownButton<_DotsColor>(
+        value: dotsBorderColor,
+        items: _DotsColor.values.map(_itemBuilderDotsColor).toList(),
+        onChanged: (value) {
+          setState(() {
+            dotsBorderColor = value!;
+          });
+        },
+      ),
+    );
+  }
+
+  Widget _buildListTileDotsSize() {
+    return ListTile(
+      title: const Text('Dots size: '),
+      trailing: DropdownButton<double>(
+        value: dotsSize,
+        items: _kDotsSizeList.map(_itemBuilderDotsDimen).toList(),
+        onChanged: (value) {
+          setState(() {
+            dotsSize = value!;
+          });
+        },
+      ),
+    );
+  }
+
+  Widget _buildListTileDotsBorderWidth() {
+    return ListTile(
+      title: const Text('Dots border width: '),
+      trailing: DropdownButton<double>(
+        value: dotsBorderWidth,
+        items: _kDotsBorderWidthList.map(_itemBuilderDotsDimen).toList(),
+        onChanged: (value) {
+          setState(() {
+            dotsBorderWidth = value!;
+          });
+        },
+      ),
+    );
+  }
+
+  Widget _buildListTileDotsShape() {
+    return ListTile(
+      title: const Text('Dots shape: '),
+      trailing: DropdownButton<BoxShape>(
+        value: dotsShape,
+        items: BoxShape.values.map(_itemBuilderDotsShape).toList(),
+        onChanged: (value) {
+          setState(() {
+            dotsShape = value!;
+          });
+        },
+      ),
+    );
+  }
+
+  Widget _buildListTileDotsSpacing() {
```

Comment:
💅 `{}` can be omitted for instant `return`:
```dart
  Widget _buildListTileDotsSpacing() => ListTile(
```
please check other places as well 🙏